crates/cargo-bsp/src/cargo_communication/request_handle.rs

```diff
@@ -22,8 +24,8 @@ use crate::cargo_communication::utils::targets_ids_to_targets_details;
 use crate::server::global_state::GlobalStateSnapshot;
 
 pub(crate) struct RequestHandle {
-    cancel_sender: Sender<Event>,
-    _thread: jod_thread::JoinHandle,
+    pub(super) cancel_sender: Sender<Event>,
+    pub(super) _thread: jod_thread::JoinHandle,
```

Comment:
czemu tutaj _thread?

Reply:
a nie thread po prostu?

Reply:
Ponieważ to pole nie jest nigdy odczytywane, jest tylko ustawiane tworząc RequestHandle